Adding --min_DEG_pathway as a parameter (#84)
qbic-pipelines/rnadeseq: docs/usage.md

Text file containing KEGG pathways codes to be excluded from pathway plotting (e.g. because kegg pathways xml contain errors in the KEGG resource).

### `--min_DEG_pathway`

> Integer indicating how many genes in a pathway must be differentially expressed to be considered as enriched, and report these pathways in tables and the final report. The default value is 1. 

Reviewer: ```suggestion
Integer indicating how many genes in a pathway must be differentially expressed to be considered as enriched, and report these pathways in tables and the final report. The default value is 1.
```

> Integer indicating how many genes in a pathway must be differentially expressed to report these pathways in tables and the final report. If a string is given, it will be ignored. The default value is 1. 

Reviewer: ```suggestion
Integer indicating how many genes in a pathway must be differentially expressed to be considered as enriched, and report these pathways in tables and the final report. The default value is 1. 
```
